packages/shorebird_cli/test/src/platform/apple_test.dart

```diff
@@ -92,6 +92,65 @@ To add macOS, run "flutter create . --platforms macos"''');
         ).thenReturn(projectRoot);
       });
 
+      group('supplement artifact', () {
```

Comment:
```suggestion
      group('copySupplementFilesToSnapshotDirs', () {
```

our convention has been to have groups for the methods you're unit testing so that when you run the test it reads like a sentence:

`copySupplementFilesToSnapshotDirs copies all files next to snapshots`

instead of:

`supplement artifact copies all files next to snapshots`